portality/settings.py

```diff
@@ -1,5 +1,9 @@
 import os
 
+DOAJ_VERSION = "2.10.1_SNAPSHOT-1"
```

Comment:
This should eventually just be the current commit - ideally it will be the last commit / timestamp when a static resource last changed, but that's too complex. My point is that then devs don't have to know / remember about bumping the version when they edit a static file, every commit will bust the cache automatically. Doesn't prevent merging this now ofc,.